imports/plugins/included/product-detail-simple/client/components/variant.js

```diff
@@ -9,6 +9,20 @@ import { ReactionProduct } from "/lib/api";
 
 import { ProductVariant } from "/lib/collections/schemas";
 
+const customStyle = {
```

Comment:
No inline styles, add this is a CSS file somewhere, most likely one of the files here `/imports/plugins/included/default-theme/client/styles/products`